packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt

```diff
@@ -24,55 +22,26 @@
 # Memory (System/GPU configuration )
 ################################################################################
 
-# default GPU memory split (do not change if you do not know what you are doing)
+# Default GPU memory split (do not change if you do not know what you are doing)
   gpu_mem=128
 
-# SDRAM size based GPU memory split, uncomment and setup to override 'gpu_mem'
+# Configure GPU memory based on SDRAM size - overrides above setting
   gpu_mem_256=112
   gpu_mem_512=160
   gpu_mem_1024=256
 
 ################################################################################
-# Overclocking settings
-# WARNING: Do not change/enable if you do not know what you are doing!
-#          The System may become unstable or you can have data corruption or
-#          you can loose your warranty if you set wrong settings
-#
-# please read: http://elinux.org/RPi_config.txt#Overclocking_configuration
+# Various settings see also: http://elinux.org/RPi_config.txt
 ################################################################################
+# For overclocking information, please read:
+# http://elinux.org/RPi_config.txt#Overclocking_configuration
 
-# Overclock mode settings.
-#
-# default recommended values are: arm_freq | core_freq | sdram_freq | over_voltage
-# no overclocking               :    700   |    250    |    400     |      0
-# mode 'Modest'                 :    800   |    300    |    400     |      0
-# mode 'Medium'                 :    900   |    333    |    450     |      2
-# mode 'High'                   :    950   |    450    |    450     |      6
-# mode 'Turbo'                  :   1000   |    500    |    600     |      6
-# mode 'Pi2'                    :   1000   |    500    |    500     |      2
-
-# arm_freq=700
-# core_freq=250
-# sdram_freq=400
-# over_voltage=0
-
-# set to 'force_turbo=1' to disable dynamic overclocking (you can lose your warranty!)
-  force_turbo=0
-
-# If you have any data corruption using Turbo Mode and overclocking
-# try with this setting uncommented
-# If this still fails, try to use Medium mode without "over_voltage=2"
-# If it ultimately keeps failing, forget about this until it gets fixed
-# Use a more conservative mode that will works fine in any cases:
-# initial_turbo=30
-
-################################################################################
-# Overscan settings
-################################################################################
+# Set 'force_turbo=1' to disable dynamic overclocking.
+force_turbo=0
 
 # Make display smaller to stop text spilling off the screen
-# see also http://www.raspberrypi.org/phpBB3/viewtopic.php?f=67&t=15700
-# overscan_scale=1
+# See also http://www.raspberrypi.org/phpBB3/viewtopic.php?f=67&t=15700
+# Note that the overscan settings only affect the splash screena and not Kodi.
```

Comment:
"screena". You could mention the calibration menu is the right place to adjust kodi. You could also mention that TV settings are best place to solve this, but that may be going too far.